media-gfx/swappy/swappy-1.2.1.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson xdg-utils
+
+DESCRIPTION="A Wayland native snapshot and editor tool, inspired by Snappy on macOS"
+HOMEPAGE="https://github.com/jtheoof/swappy"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/jtheoof/swappy"
+else
+	SRC_URI="https://github.com/jtheoof/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+
+LICENSE="MIT"
+SLOT="0"
+IUSE="+libnotify +clipboard man"
+
+DEPEND="
+	dev-libs/glib:2
+	x11-libs/gtk+:3
+	x11-libs/cairo
+	x11-libs/pango
+	media-fonts/fontawesome[otf]
+	clipboard? ( gui-apps/wl-clipboard )
```

Comment:
This is an optional runtime dependency.
This should not be a USE flag, instead look at the `optfeature.eclass` and in `pkg_postinst` to give an message saying that wl-clipboard can be used for optional clipboard support.

Reply:
TIL!